portfolio/src/main/webapp/fun-features.html

```diff
@@ -9,7 +9,7 @@
     <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
-  <body onload="createMap(); getComment()">
+  <body onload="createMap(); getComment(); checkLogin()">
```

Comment:
This is up to personal preference but it might be cleaner to have the onload function in the Javascript file like this: window.onload = function() {  ... }

Reply:
ack